astropy/wcs/src/pyutil.c

```diff
@@ -883,7 +886,7 @@ set_pvcards(
 
   for (i = 0; i < size; ++i)
   {
-    if (!PyArg_ParseTuple(PySequence_Fast_GET_ITEM(value, i), "iid",
+    if (!PyArg_ParseTuple(PySequence_GetItem(value, i), "iid",
```

Comment:
This isn't quite the same.

`PySequence_Fast_GET_ITEM` returns a borrowed reference while `PySequence_GetItem` returns an owned reference. So you're now missing a `Py_DECREF`.

Reply:
Interesting. Grepping for `PySequence_GetItem` in our existing code I don't see any `Py_DECREF` coming along. Does it need fixing everywhere ?